crates/hdk/src/error.rs

```diff
@@ -21,6 +22,9 @@ pub enum HdkError {
 
     #[error("A remote zome call was made but there was a network error: {0}")]
     ZomeCallNetworkError(String),
+
+    #[error(transparent)]
+    Infallible(#[from] Infallible),
```

Comment:
Any reason for this in this PR? Seems unrelated?

Reply:
This is also due to updating syn/proc-macro2; new error type produced